spring-rabbit/src/main/java/org/springframework/amqp/rabbit/AsyncRabbitTemplate.java

```diff
@@ -71,15 +74,15 @@
  * @since 1.6
  */
 public class AsyncRabbitTemplate implements SmartLifecycle, MessageListener, ReturnCallback, ConfirmCallback,
-		BeanNameAware {
+		BeanNameAware, AsyncAmqpTemplate {
```

Comment:
Nit-pick, but `AsyncAmqpTemplate` is the main contract of this class. So, it should be first in the `implements` declaration. :smile: